src/Solver/Numerov.h

```diff
@@ -0,0 +1,57 @@
+#ifndef NUMEROV_H
+#define NUMEROV_H
+
+#define __STDCPP_WANT_MATH_SPEC_FUNCS__ 1
+
+#define dx 0.01
```

Comment:
Always talking about a common style, shouldn't constant values be uppercase and use snakecase?
e.g. DX, ERR, PI, HBAR, MASS

Reply:
Sure. The plan is to create a [World class](https://github.com/Scienza/Schroedinger/wiki/Classes), those constants shouldn't exist in the future, it's temporary.

Reply:
Ok, all right then